Themis/Models/Repository.swift

```diff
@@ -28,6 +28,9 @@ class Node: Hashable, ObservableObject {
     let type: FileType
     var children: [Node]?
     @Published var code: String?
+    var templateCode: String?
```

Comment:
Why is this not `@Published`?

Reply:
Because it is not observed by any view, making it unnecessary to publish.